Tests/StackClusterTests.swift

```diff
@@ -112,7 +112,7 @@ class StackClusterTests: QuickSpec {
                     labelComponent(title: "1", color: .red, labelSize: size),
                     labelComponent(title: "2", color: .red, labelSize: size),
                     labelComponent(title: "3", color: .red, labelSize: size),
-                    labelComponent(title: "4", color: .red, labelSize: size),
+                    labelComponent(title: "4", color: .red, labelSize: size)
```

Comment:
I am fine with both solution, but I would choose the one that differs not from the default ruleset, so we can avoid custom rules wherever it is possible.